dataset/jeopardy_dataset.py

```diff
@@ -0,0 +1,275 @@
+import os.path
+import json
+from PIL import Image
+from torch.utils.data import Dataset
+import torch
+import os
+import string
+from tqdm import tqdm
+import nltk
+
+# START_TOKEN = "<s>"
+# END_TOKEN = "</s>"
+PAD_TOKEN = "<pad>"
+UNK_TOKEN = "<unk>"
+
+class JeopardyDataset(Dataset):
+    def __init__(self, 
+        questions_file, 
+        answers_file, 
+        images_dir, 
+        transform, 
+        word2idx=None, 
+        train=True, 
+        q_len=8, 
```

Comment:
why 8? default roberta encoding is like 512 if i remember correctly

Reply:
can we talk about this in our meeting tomorrow

Reply:
also might be worth it to get rid of q_len constraint entirely, I'll look at the frequency of various question lengths tomorrow